src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/domene/BehandlingRepository.kt

```diff
@@ -46,10 +49,54 @@ interface BehandlingRepository : JpaRepository<Behandling, Long> {
     @Query(
         """select b from Behandling b
                            inner join BehandlingStegTilstand bst on b.id = bst.behandling.id
-                        where b.fagsak.id = :fagsakId AND bst.behandlingSteg = 'BESLUTTE_VEDTAK' AND bst.behandlingStegStatus IN (no.nav.familie.ks.sak.kjerne.behandling.steg.BehandlingStegStatus.KLAR, no.nav.familie.ks.sak.kjerne.behandling.steg.BehandlingStegStatus.VENTER)"""
+                        where b.fagsak.id = :fagsakId AND bst.behandlingSteg = 'BESLUTTE_VEDTAK' 
+                        AND bst.behandlingStegStatus IN (no.nav.familie.ks.sak.kjerne.behandling.steg.BehandlingStegStatus.KLAR, 
+            no.nav.familie.ks.sak.kjerne.behandling.steg.BehandlingStegStatus.VENTER)"""
     )
     fun finnBehandlingerSendtTilGodkjenning(fagsakId: Long): List<Behandling>
 
     @Query("SELECT b FROM Behandling b JOIN b.fagsak f WHERE f.id = :fagsakId AND b.status = 'AVSLUTTET' AND f.arkivert = false")
     fun finnByFagsakAndAvsluttet(fagsakId: Long): List<Behandling>
+
+    @Query(
+        value = """WITH sisteiverksattebehandlingfraløpendefagsak AS (
+                            SELECT f.id AS fagsakid, MAX(b.opprettet_tid) AS opprettet_tid
+                            FROM behandling b
+                                   INNER JOIN fagsak f ON f.id = b.fk_fagsak_id
+                                   INNER JOIN tilkjent_ytelse ty ON b.id = ty.fk_behandling_id
+                            WHERE f.status = 'LØPENDE'
```

Comment:
Hvis ikke alt gjort, så bør man nok få noen indexer

Reply:
@stigebil Hvilke parameterne  er en god kandidat for indexer, tror du? Vi har allerede indeks på fagsakId i behandling tabell, behandlingId og utbetalingsoppdrag på tilkjent_ytelse.

Reply:
index på utbetalingsoppdrag IS NOT NULL har vi allerede. Jeg legger til indekser for status på fagsak og opprettet_tid på behandling tabell.